src/ParameterType.php

```diff
@@ -41,6 +41,11 @@ final class ParameterType
      */
     public const BINARY = 16;
 
+    /**
+     * Represents an ascii string data type
+     */
+    public const ASCII_STRING = 17;
```

Comment:
No, this should stay `ASCII`. This is a binding type that could be reused by different data types (e.g. `'ascii_text'`). The `'ascii_string'` is a data type.